jobboard/migrations/0003_job_work_from.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.9 on 2021-11-22 13:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('jobboard', '0002_fixed_data'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
I think it's better to create a new migration file that creates the table from scratch with this field instead of having a separate file just to add it